muted-tests.yml

```diff
@@ -97,8 +97,6 @@ tests:
 - class: org.elasticsearch.search.StressSearchServiceReaperIT
   method: testStressReaper
   issue: https://github.com/elastic/elasticsearch/issues/115816
-- class: org.elasticsearch.xpack.application.connector.ConnectorIndexServiceTests
-  issue: https://github.com/elastic/elasticsearch/issues/116087
```

Comment:
What about unmuting the tests?

Reply:
It got muted due couple of flakey runs in the past ... I checked the logs and it seemed to be weird ES error. It has been running for almost a year now fine